utils/firebase/storage.js

```diff
@@ -0,0 +1,30 @@
+import firebase from "./firebase";
+import uuid from "uuid";
+export async function uploadImageAsync(uri) {
```

Comment:
Not really sure what all of this does, but if it works then good with me

Reply:
same i just moved the firebase stuff in image upload to its own file